### ISSUE TITLE
Can you save sections of the image that are masked after using SegAutoMaskPredictor()

### ISSUE THREAD

**Issue by ksd3** [kadirnar/segment-anything-video#83]
I can't find this in the documentation. After running results = SegAutoMaskPredictor().image_predict(
    source="firststeve.png",
    model_type="vit_h", # vit_l, vit_h, vit_b
    points_per_side=4, 
    points_per_batch=16,
    min_area=0,
    output_path="output.png",
    show=False,
    save=True,
)

I can save the segmented image, but I can't pull out the individual segmented pieces. Does this package support doing something like this or do I have to use the original repository? I just want to know if something like this exists:
![image](https://github.com/kadirnar/segment-anything-video/assets/89213519/57d980df-4587-4b2d-b110-1c7cc93e2aad)

**Comment by kadirnar** [comment]
Hi, do you want to save every segmented object?

**Comment by ksd3** [comment]
Yes, I want to save every segmented object. I know this can be done (tediously) by inspecting each ``bbox`` variable (that is the result of SamAutomaticMaskGenerator from the original repository) and cropping the image based on the ``xywh`` format but I wondered if there was a way to do it faster using metaseg.

**Comment by kadirnar** [comment]
I will add this feature but I am busy right now. It may take some time to add. If you want to add, I can help.
